devHelper/commands/arma3.json

```diff
@@ -209,9 +217,6 @@
         "boundingCenter",
         "brakesDisabled",
         "break",
-        "breakOut",
-        "breakTo",
-        "breakWith",

```

Comment:
something seems off, those shouldn't be removed. I'll have to check the python script before approving.

Reply:
forgetfich wuhu